openlibrary/catalog/amazon/import.py

```diff
@@ -166,7 +166,7 @@ def import_file(filename):
             # TODO load book
             continue
         #print asin, found
-        #print `edition['title'], edition.get('subtitle', None), edition.get('flags', None), edition.get('binding', None)`
+        #print(repr(edition['title'], edition.get('subtitle', None), edition.get('flags', None), edition.get('binding', None)))
```

Comment:
Many of these commented out print statements should probably be removed.
Unless the code is in the scripts folder I'd suggest we remove all debugging `prints` commented out or not. If the message is important it should be passed to logging. Maybe that's for a different issue (or while people are working in an area)

Reply:
Definitely a separate issue. This should be: a) part of the coding guidelines, and b) the developer guidelines should say to bring code being worked on up to standard with the coding guidelines (in a separate commit!).

I don't think it's a good idea to go through and do a wholesale cleanup.

Reply:
Yes so I wasn't sure if what to do with such print statements. Which are commented out! Maybe we could look at module wise review of removing debugging prints?